ocaml/tests/suite_alcotest.ml

```diff
@@ -75,4 +75,5 @@ let () =
       @ Test_pvs_cache_storage.tests
       @ Test_extauth_plugin_ADpbis.tests
       @ Test_helpers.tests
-      @ Test_datamodel_utils.tests);
+      @ Test_datamodel_utils.tests
+      @ Test_psr.tests );
```

Comment:
It doesn't matter much here, but `List.concat` would be more efficient. We also had some nasty surprised when using `@` together with if-then-else.